docker/reaper/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:16.04
+
+# use a common app path, copied from python-onbuild:latest
+ENV WORKDIR /usr/src/app
+RUN mkdir -p ${WORKDIR}
+WORKDIR ${WORKDIR}
+
+# install Java
+RUN apt-get update \
+    && apt-get install -y \
+        openjdk-8-jdk
+
+# grab source
+COPY ./packages/*.jar ${WORKDIR}/
```

Comment:
This line fails on me each time I try to run `docker-compose build` : 
```
 ---> Using cache
 ---> 0505ac0bcc1f
Step 6/9 : COPY ./packages/*.jar ${WORKDIR}/
ERROR: Service 'reaper' failed to build: No source files were specified
```

The packages directory has files in it : 
```
MacBook-Pro-de-Alexander:cassandra-reaper adejanovski$ ls packages
cassandra-reaper-0.4.0-SNAPSHOT.jar		reaper-0.4.0_SNAPSHOT-1.x86_64.rpm
original-cassandra-reaper-0.4.0-SNAPSHOT.jar	reaper_0.4.0-SNAPSHOT_amd64.deb
```

Do you know what's wrong here ?